database/schema.sql

```diff
@@ -0,0 +1,112 @@
+DROP TABLE IF EXISTS subscriber_county_assignment, subscriber, forecast, solar_feature, county, body_assignment, body, constellation, region, aurora_alert, aurora_colour, image;
```

Comment:
Dropped tables when the sql script is run, that the reset.zsh script will run later. LGTM.